git_pull/run.sh

```diff
@@ -43,7 +43,7 @@ while true; do
 
     # Enable autorestart of homeassistant
     if [ "$AUTO_RESTART" == "true" ]; then
-        changed_files="$(git diff-tree -r --name-only --no-commit-id ORIG_HEAD HEAD)"
+        changed_files="$(git diff-tree -r --name-only --no-commit-id HEAD@{1} HEAD)"
```

Comment:
You can fix the lint with `# shellcheck disable=` before your statement

Reply:
I don't understand how to do this. can you show me where exactly the statement goes.

Reply:
use `'HEAD@{1}'`